src/meltano/core/tracking/schemas/plugins_context.json

```diff
@@ -0,0 +1,117 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a Snowplow context describing the plugins being used by Meltano",
+    "self": {
+        "vendor": "com.meltano",
+        "name": "plugins_context",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "context_uuid": {
+            "description": "A UUID that uniquely identifies an instance of this context",
+            "type": "string",
+            "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+            "minLength": 36,
+            "maxLength": 36
+        },
+        "plugins": {
+            "description": "An array containing information about each plugin in order",
+            "type": "array",
+            "items": {
+                "description": "",
+                "type": "object",
+                "properties": {
+                    "category": {
+                        "description": "",
+                        "type": "string",
+                        "enum": [
+                            "extractor",
+                            "loader",
+                            "trasformer"
+                        ]
+                    },
+                    "type": {
+                        "description": "",
+                        "type": "string",
+                        "maxLength": 256
+                    },
+                    "name_hash": {
+                        "description": "",
```

Comment:
From zoom: desc should describe "this is a 64char hash field" , but drop pattern, max, min constraints.